### ISSUE TITLE
Add alias exit key for SIGINT CTRL+C

### ISSUE THREAD

**Issue by Steveland** [ytdl-org/youtube-dl#27816]
## Checklist

- [x] I'm reporting a feature request
- [x] I've verified that I'm running youtube-dl version **2021.01.08**
- [x] I've searched the bugtracker for similar feature requests including closed ones

## Description

Is it possible to add a simple alias key (for example the 'q' key) to exit youtube-dl properly during livestream recordings. And display a message "Press [q] to stop" like in ffmpeg.exe.

I ask this because in Windows, GenerateConsoleCtrlEvent CTRL_C_EVENT does not seem to work as expected, we have to use CTRL_BREAK_EVENT instead and it makes ffmpeg.exe to close before it can perform post-processing.

**Comment by dstftw** [comment]
What do you mean by "does not seem to work as expected"? Post the code you're using to run youtube-dl and send event.

**Comment by garoto** [comment]
`ctrl+c` in a cmd.exe window makes ffmpeg shutdown gracefully for me when dumping HLS streams. I'm on Windows 8.1.

**Comment by Steveland** [comment]
Thank you Sergey and Garoto for replying.

Yes, Ctrl+C also worked fine in cmd.exe for me. But it did not work when trying to close from c++ app (youtube-dl.exe being created as a child process) using GenerateConsoleCtrlEvent.

Anyway, I finally solved the issue. The problem was the CREATE_NEW_PROCESS_GROUP flag in CreateProcess, it only works for Ctrl+Break event, so I had to remove it for Ctrl+C to work with GenerateConsoleCtrlEvent CTRL_C_EVENT.

After this I had another issue, GenerateConsoleCtrlEvent was raising an exception. I found out it was just a normal behaviour in Microsoft Visual Studio debug mode, and it can be turned off in Debug menu/Exceptions/Win32 Exceptions.
